sdk/js/src/cosmwasm/query.testnet.test.ts

```diff
@@ -0,0 +1,829 @@
+  );
+  expect(tbQueryResult?.admin).toEqual(
+    CONTRACTS.TESTNET.injective.token_bridge
+  );
+});
+test.skip("testnet - injective query guardian_set_info", async () => {
+  const network = getNetworkInfo(getEndPoint());
+  const client = new ChainGrpcWasmApi(network.sentryGrpcApi);
+  // https://k8s.testnet.lcd.injective.network/cosmwasm/wasm/v1/contract/inj1xx3aupmgv3ce537c0yce8zzd3sz567syuyedpg/smart/eyJndWFyZGlhbl9zZXRfaW5mbyI6e319
+  const queryResult = await client.fetchSmartContractState(
+    CONTRACTS.TESTNET.injective.core,
+    Buffer.from('{"guardian_set_info":{}}').toString("base64")
+  );
+  let result: any = null;
+  if (typeof queryResult.data === "string") {
+    result = JSON.parse(
+      Buffer.from(queryResult.data, "base64").toString("utf-8")
+    );
+  }
+  expect(result?.guardian_set_index).toEqual(0);
+  expect(result?.addresses.length).toEqual(1);
+});
+test.skip("testnet - injective query state", async () => {
+  const network = getNetworkInfo(getEndPoint());
+  const client = new ChainGrpcWasmApi(network.sentryGrpcApi);
+  const queryResult = await client.fetchSmartContractState(
+    CONTRACTS.TESTNET.injective.core,
+    Buffer.from('{"get_state":{}}').toString("base64")
+  );
+  let result: any = null;
+  if (typeof queryResult.data === "string") {
+    result = JSON.parse(
+      Buffer.from(queryResult.data, "base64").toString("utf-8")
+    );
+  }
+  expect(result?.fee?.denom).toEqual("inj");
+  expect(result?.fee?.amount).toEqual("0");
+});
+test.skip("testnet - injective query token bridge", async () => {
+  const network = getNetworkInfo(getEndPoint());
+  const client = new ChainGrpcWasmApi(network.sentryGrpcApi);
+  // const wrappedAsset = await getIsWrappedAssetInjective(
+  //   CONTRACTS.TESTNET.injective.token_bridge,
+  //   "inj10jc4vr9vfq0ykkmfvfgz430w8z6hwdlqhdw9l8"
+  // );
+  // console.log("isWrappedAsset", wrappedAsset);
+  const queryResult = await client.fetchSmartContractState(
+    CONTRACTS.TESTNET.injective.token_bridge,
+    Buffer.from(
+      JSON.stringify({
+        wrapped_registry: {
+          chain: coalesceChainId("injective"),
+          address: tryNativeToHexString(
+            "inj10jc4vr9vfq0ykkmfvfgz430w8z6hwdlqhdw9l8",
+            "injective"
+          ),
+        },
+      })
+    ).toString("base64")
+  );
+  let result: any = null;
+  if (typeof queryResult.data === "string") {
+    result = JSON.parse(
+      Buffer.from(queryResult.data, "base64").toString("utf-8")
+    );
+    console.log("result", result);
+  }
+});
+test.skip("testnet - injective attest native asset", async () => {
+  const tba = CONTRACTS.TESTNET.injective.token_bridge;
+  const wallet = "inj180rl9ezc4389t72pc3vvlkxxs5d9jx60w9eeu3";
+  const result = await attestFromInjective(tba, wallet, "inj");
+  console.log("result", result.params.msg);
+  console.log("token", JSON.stringify(result.params.msg));
+  console.log("json", result.toJSON());
+  const walletPKHash =
```

Comment:
I think this was public, but change all these to   `const walletPKHash = process.env.ETH_KEY || "";`

Reply:
Done